src/libbson/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ if (ENABLE_TESTS AND NOT ENABLE_STATIC MATCHES "ON|AUTO")
    message (FATAL_ERROR "-DENABLE_STATIC=OFF also requires -DENABLE_TESTS=OFF")
 endif ()
 
+set(BSON_OUTPUT_BASENAME "bson" CACHE STRING "Output bson library base name")
```

Comment:
Please add a space here -> `set (` for stylistic consistency

Reply:
done